package.lisp

```diff
@@ -0,0 +1,22 @@
+(defpackage :cl-data-structures
+  (:use :common-lisp)
```

Comment:
package should be inside src folder; exported symbols do not need to be quoted nor keyworded.